src/lib/rules/player.hh

```diff
@@ -60,6 +61,19 @@ struct Players final : utils::IBufferizable
         }
     }
 
+    std::string scores_yaml() const
+    {
+        std::stringstream ss;
+        for (auto& player : players)
+        {
+            ss << "---" << std::endl
+               << "player: " << player->name.c_str() << std::endl
```

Comment:
how nasty (how user controlled) can this name be? does it need escaping and/or quoting at least?

Reply:
It's a free string (incl. \0 I think). A rogue client could potentially send a specially crafted name to overwrite their score here.

Reply:
Yeah or naming their champion 123 without bad intentions, which converts to an int, which is not what we want.